runelite-client/src/main/java/net/runelite/client/plugins/vorkath/VorkathPlugin.java

```diff
@@ -132,6 +132,11 @@ public void onAnimationChanged(AnimationChanged event)
 			case AnimationID.VORKATH_DEATH:
 				reset();
 				break;
+			case AnimationID.VORKATH_SLASH_ATTACK:
+				normalAttackCounter++;
```

Comment:
Probably move these 3 lines to separate method and call it from both here and onProjectileMoved